### ISSUE TITLE
parse html doesn't parse template tag

### ISSUE THREAD

**Issue by Aliaksandr-Padabed** [JPeer264/node-rcs-core#58]
``` html
<div class="will-be-parsed">
  <template type="some_type">
    <div class="will-not-be-parse"></div>
  </template>
</div>
```
Parse5 handles template tag differently, children of template tag are placed in to content property.

**Comment by JPeer264** [comment]
Thanks for your issue. That is interesting. I'm investigating on this.

**Comment by JPeer264** [comment]
@Aliaksandr-Padabed should be fixed in `v2.3.1`
